Makefile

```diff
@@ -125,6 +126,33 @@ load-mongodb:
 		-p recordcount=1000000 \
 		-p mongodb.url=mongodb://localhost:27017/ycsb
 
+start-memcached:
+	@echo "Starting memcached server..."
+	@memcached -d -l 0.0.0.0 -p $(MEMCACHED_PORT) -u root
+	@sleep 2
+	@echo "Verifying memcached is running..."
+	@if pgrep memcached > /dev/null; then \
+		echo "Memcached process found"; \
+		# netstat -nltp | grep $(MEMCACHED_PORT); \
+	else \
+		echo "Failed to start memcached"; \
+		exit 1; \
+	fi
+
+benchmark-memcached:
+	@${MAKE} start-memcached
+	@python python/kernmlops collect -v \
+		-c ${KERNMLOPS_CONFIG_FILE} \
+		--benchmark memcached
+
+load-memcached:
```

Comment:
is this required every time docker is invoked or is it a 1 time cost like mongodb?

Reply:
Yeah you are right, I will add it to be a 1-time cost.